pkg/cliplugins/api/cmd/cmd.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+
+	"github.com/kcp-dev/kcp/pkg/cliplugins/api/plugin"
+)
+
+var (
+	// TODO: add other examples related to permission claim commands.
+	bindExampleUses = `
+	# binds to the mentioned workspace in the command, e.g the below command will create
+	# an APIBinding named "my-binding", that binds to "my-export" present in "root:my-export-service" workspace.
+	%[1]s api bind my-export --path root:my-export-service --name my-binding
+	`
+)
+
+func New(streams genericclioptions.IOStreams) (*cobra.Command, error) {
+	cmd := &cobra.Command{
+		Use:              "api",
+		Short:            "Operations related to binding with API",
+		SilenceUsage:     true,
+		TraverseChildren: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return cmd.Help()
+		},
+	}
+
+	bindOpts := plugin.NewBindOptions(streams)
+	bindCmd := &cobra.Command{
+		Use:          "bind [<APIExport-name>]",
```

Comment:
this can be also absolute? How do we distinguish catalog entries later?

Would have expected something like: `kubectl kcp bind apiexport foo`, `kubectl kcp bind catalogentry foo`

Do we have a gdoc where we explore the space?

Reply:
The issue had `kubectl kcp apis bind|claim|whatever` - I don't know that we want `bind` top-level, perhaps?

Reply:
modified this to be: `kubectl-kcp bind apiexport root:my-export-service:my-export` based on the discussion in design doc.